### ISSUE TITLE
Copy of Design | CH 36 | Replace Wizard Pattern

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68008]
## Issue Description
_CAIA #63702 recommended that the depreciated wizard pattern be replaced by the following suggested solution:_

### Long-term solution
As mentioned in the Design System [wizard pattern](https://design.va.gov/patterns/wizards) page, the [Wizard pattern is now deprecated](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399) and replaced by the the [help users complete a sub-task pattern](https://design.va.gov/patterns/help-users-to/complete-a-sub-task). 

There are many accessibility defects related to the wizard pattern, as described [here](https://design.va.gov/patterns/wizards#retrofit-this-pattern).

We highly recommend that you use the sub-task pattern instead of the wizard pattern to avoid this accessibility issue and reduce the user's cognitive load.

## References, articles, or WCAG support

1. [Focus management example](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start)
2. Reduce cognitive load by having [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page)
3. [How to retrofit the wizard pattern, if you absolutely must keep it](https://design.va.gov/patterns/wizards#research-and-accessibility-rationale)
---
## Tasks
- [ ] Create design solution to replace the depreciated wizard element
- [ ] Obtain CAIA approval

## Acceptance Criteria
- [ ] Sketch file of solution
- [ ] Assign to Laura who will create follow-on engineering ticket
---

**Comment by fiorella-io** [comment]
We must design a new experience for this issue since the current design uses a deprecated approach. If we're using T-shirt size, I would say this issue is a Medium, if we're using Fibonacci it'll be a 6

**Comment by fiorella-io** [comment]
We need to figure out what to do with the VR&E orientation slide to replace this wizard. If we want to keep it or not. I asked a few questions on [this thread](https://dsva.slack.com/archives/C057W3N9K2S/p1703095933717989). 

It is currently embedded in this depreciated wizard pattern before starting the Ch31 online. The way it was implemented has a lot of accessibility issues. If we want to keep it we need to decide where it lives, but it has to live on its own page.

**Comment by fiorella-io** [comment]
Created flows of the wizard and created designs that were added to this ticket.
